src/main.py

```diff
@@ -1,17 +1,44 @@
-from typing import Optional
-
-
-def print_something(something: Optional[str]) -> None:
-    print(something)
-
-
-def main():
-    print_something("Welcome to ingeSTation project!")
-
-
-def ingestation():
-    return main()
+# from modules.configs import MergedConf
+# from modules.misc.arguments import CliInput
+# from modules.misc import utils
+# from modules.outputs import blocks, logging
+import logging
+import os
+
+from modules.configs import config
+from modules.global_vars import global_vars
+
+# def main(cli_options: dict) -> None:
+#     init_params = utils.InitParams()
+#     loggers = logging.Loggers(cli_options)
+#     logger = loggers.logger
+#     log_blocks = blocks.Blocks(logger)
+#     log_blocks.log_starting_messages(init_params, cli_options)
+#
+#     config = MergedConf(logger, cli_options)
+#     log_blocks.log_loaded_configs(config)
+#
+#
+# def cargoloader():
+#     return main(CliInput.parse_arguments(CliInput()))
 
 
 if __name__ == "__main__":
-    ingestation()
+    app_conf = config.Conf("", "application")
```

Comment:
These are not needed here, merged_conf/final_conf will suffice in main(). configs module and MergedConf class should encapsulate all config functionalities.